packages/safe-tools-client/app/components/block-explorer-button/index.css

```diff
@@ -0,0 +1,36 @@
+.block-explorer-button--disabled {
+  position: relative;
+  pointer-events: unset !important;
+
+  --boxel-button-color: transparent !important;
+  --boxel-button-border: 1px solid var(--boxel-purple-200) !important;
+  --boxel-button-text-color: var(--boxel-purple-300) !important;
+}
```

Comment:
Is there an opportunity to improve BoxelButton rather than having to do custom styling for this?

Reply:
I updated this to use cssVar to do the styling. https://github.com/cardstack/cardstack/pull/3644/commits/fa4912045bb816562332767ada99d6a1ab550a82#diff-b08a86a460d3c830173b2f4d7e0708975a275c21930d7cbe0ff072593ea9a996R50

Reply:
I've improved the BoxelButton by adding a tooltip argument.